diracx-logic/src/diracx/logic/jobs/status.py

```diff
@@ -474,3 +476,41 @@ async def remove_jobs_from_task_queue(
         await recalculate_tq_shares_for_entity(
             owner, owner_group, vo, config, task_queue_db
         )
+
+
+async def set_job_parameters_or_attributes(
+    updates: dict[int, dict[str, Any]],
+    job_db: JobDB,
+    job_parameters_db: JobParametersDB,
+):
+    """Set job parameters or attributes for a list of jobs."""
+    possible_attribute_columns = [
+        col.name.lower() for col in _get_columns(Jobs.__table__, None)
+    ]
```

Comment:
Is this (`lower()` of all column names) and the subsequent check needed? Can't we compare without lowering?

Reply:
I'm inclined to agree. I just wonder what would happen if somebody passed a case-sensitive un-matching attribute column that then gets registered in the JobParameterDB instead instead of throwing an error. Maybe we should throw an exception if the case-sensitive vs. insensitive matches don't line up

Reply:
I see, nothing is perfect. Do as you feel.

Reply:
done